src/core/createLogger.js

```diff
@@ -18,6 +18,7 @@ const createLogger = (window, logController, prefix) => {
   };
 
   return {
+    enabled: logController.logEnabled,
```

Comment:
When `logController.logEnabled` changes, `logger.enabled` won't change. To make `logger.enabled` reflect the latest value, you'll need to change this to:

```
get enabled() {
  return logController.logEnabled;
},
```

I recommend adding a test for this scenario inside `createLogger.spec.js`.